### PR TITLE
Update rooms-authentication.md

### PR DESCRIPTION
**Pull request by kdspiess**
@sohailta - Please review the proposed changes to the Azure AD conditional access section.   

@dstrome - These changes aren't quite ready for release as I would like to gather some TAP customer feedback for the next couple of weeks prior to publishing to the publick.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/rooms/rooms-authentication.md [hunk 1]
```diff
@@ -103,6 +103,18 @@ If your setup doesn't allow for hybrid modern authentication or you need to remo
 
 ### Azure AD conditional access
 
-You can configure a resource account used with Microsoft Teams Rooms for IP/location-based access. To learn more, see [Conditional Access: Block access by location](/azure/active-directory/conditional-access/howto-conditional-access-policy-location).
+You can create a Conditional Access policy that supports IP/location-based access or that requires the device to be compliant per MDM compliance policies for the Microsoft Teams Rooms resource accounts.  For devices to support Require compliant device Conditional Access polices, the following minimum requirements must be met:
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-authentication.md:106]:
```suggestion
You can create a Conditional Access policy that supports IP/location-based access or that requires the device to be compliant per MDM compliance policies for the Microsoft Teams Rooms resource accounts. For devices to support Require compliant device Conditional Access policies, the following minimum requirements must be met:
```

File: Teams/rooms/rooms-authentication.md [hunk 2]
```diff
@@ -103,6 +103,18 @@ If your setup doesn't allow for hybrid modern authentication or you need to remo
...
 
-No other conditional access policies are supported. For more information about device compliance, see [Managing Teams Meeting Rooms with Intune](https://techcommunity.microsoft.com/t5/intune-customer-success/managing-teams-meeting-rooms-with-intune/ba-p/1069230).
\ No newline at end of file
+- Microsoft Teams Rooms application 4.8.19.0 or above
+- Windows 10 version 20H2 (10.0.19042)
+
+In addition to the minimum requirements, the following Cloud App must be excluded from teh Require compliant device Conditional Access policy:
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-authentication.md:111]:
```suggestion
In addition to the minimum requirements, the following cloud app must be excluded from the _Require compliant device Conditional Access_ policy:
```

File: Teams/rooms/rooms-authentication.md [hunk 3]
```diff
@@ -103,6 +103,18 @@ If your setup doesn't allow for hybrid modern authentication or you need to remo
...
+
+- Skype for Business Online
+
+> [!NOTE]
+> Microsoft Teams Rooms does not support the device compliance check for Skype for Business Online authentication.  If Skype for Business Online sign-in is not required for your Microsoft Teams Rooms deployment, the exclusion from the Conditional Access policy is not required.
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-authentication.md:116]:
```suggestion
> Microsoft Teams Rooms does not support the device compliance check for Skype for Business Online authentication.  If Skype for Business Online sign-in is not required for your Microsoft Teams Rooms deployment, the exclusion from the _Conditional Access policy_ is not required.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestions.

**Review by JohanFreelancer9 — APPROVED** [review]
Copy review.

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tiburd** [comment]
Article has been updated; in context, the changes in this PR appear to no longer be applicable. Closing PR.

**Pull request closed without merging**